modules/ROOT/pages/available-policies.adoc

```diff
@@ -18,8 +18,8 @@ image::policy-filters.png[]
 
 == Policy Categories
 
-The following table lists the provided policy name, the category and the characteristic the policy fulfills. +
-You can only apply one policy of each characteristic.
+Policies are categorized by the function they perform. For example, because you are able to resolve issues with the help of logs, the Message Logging policy is categorized as a troubleshooting policy. The following table lists the default policies by its category and the function it performs. 
```

Comment:
```suggestion
Policies are categorized by the function they perform. For example, because you can resolve issues with the help of logs, the Message Logging policy is categorized as a troubleshooting policy. The following table lists the default policies by each policy's category and the function it performs.
```

Reply:
This page is not needed. So removed it from the navigation.